tests/migration_test.go

```diff
@@ -375,6 +442,41 @@ var _ = Describe("Migrations", func() {
 				tests.WaitForVirtualMachineToDisappearWithTimeout(vmi, 240)
 
 			})
+			It("should be able successfully cancel a migration", func() {
+
+				vmi := tests.NewRandomVMIWithEphemeralDisk(tests.ContainerDiskFor(tests.ContainerDiskCirros))
```

Comment:
should this VMI add run that stress test to ensure the migration can't complete?  Otherwise it seems like we'd be fighting a race condition here to cancel the migration before it potentially finishes underneath us.

Reply:
Done